src/index.ts

```diff
@@ -1,4 +1,16 @@
-console.log("[ITEM-STORE] Starting item store...");
+import { logger } from "./services/logger";
+
+const logModel = {
```

Comment:
ideally this should be defined once and for all in `services/logger`, as opposed to in every file.
So we define the service name only once. The service name shouldn't be hard coded, there's actually a SERVICE_NAME variable you can get from the `./config.ts` file

also, with this technique, it seems that the date is the date the service starts, not the date when the log is created, right? We'd need each log to have its own timestamp so we can sort logs by date